XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/BaseController.cs

```diff
@@ -255,7 +255,7 @@ internal async Task TryRenderControllerModelAsync(Type controllerType, byte[] gl
                 //If the model was a prefab
                 else
                 {
-                    var controllerObject = UnityEngine.Object.Instantiate(controllerModel, MixedRealityToolkit.CameraSystem?.CameraRig.PlayspaceTransform);
+                    var controllerObject = UnityEngine.Object.Instantiate(controllerModel, MixedRealityToolkit.CameraSystem?.MainCameraRig.PlayspaceTransform);
```

Comment:
We should check if the camera system is null here and fall back to `CameraCache.Main.parent`